### PR TITLE
Allow size() to accept Array[File]

### PR DESCRIPTION
**Pull request by cjllanwarne**
Allow the `size` function to accept an array of files as input

Here's the change [in-situ](https://github.com/cjllanwarne/wdl/blob/cjl_size_of_file_array/versions/draft-3/SPEC.md#float-sizefile-string).

### CODE REVIEW

File: versions/draft-3/SPEC.md [hunk 1]
```diff
@@ -2359,6 +2359,12 @@ task example {
 Supported units are KiloByte ("K", "KB"), MegaByte ("M", "MB"), GigaByte ("G", "GB"), TeraByte ("T", "TB") as well as their [binary version](https://en.wikipedia.org/wiki/Binary_prefix) "Ki" ("KiB"), "Mi" ("MiB"), "Gi" ("GiB"), "Ti" ("TiB").
 Default unit is Bytes ("B").
 
+### Acceptable compound input types
+Varieties of the `size` function also exist for the following compound types. The `String` unit is always treated the same as above. Note that to avoid numberical overflow, very long arrays of files should probably favor larger units.
```
Comment (mcovarr) [inline versions/draft-3/SPEC.md]: Yikes having the default unit be bytes will undoubtedly...byte someone for "numberical" overflow

**Comment by skashin** [comment]
This would be very useful to have this function accept Array[File] as an argument, as I often have tasks that merge multiple inputs localized via an Array[File] declaration

**Comment by geoffjentry** [comment]
There's been almost no discussion here so I'm opening this to voting active.

**Comment by geoffjentry** [comment]
:+1:

**Comment by ambrosejcarr** [comment]
👍

**Comment by orodeh** [comment]
👍

**Comment by vdauwera** [comment]
👍

**Comment by dheiman** [comment]
👍

**Comment by geoffjentry** [comment]
This passes 5-0. I'm waiting until there's an implementation

File: versions/draft-3/SPEC.md (hunk 1 above)
Comment (yfarjoun) [reply versions/draft-3/SPEC.md]: just got bitten. @mcovarr ^^

**Pull request merged**